src/main/kotlin/edu/cornell/cs/apl/viaduct/syntax/Protocol.kt

```diff
@@ -24,3 +24,13 @@ interface Protocol : Name {
     override val nameCategory: String
         get() = "protocol"
 }
+
+/**
+ * A protocol specialized to a particular hostTrustConfiguration.
```

Comment:
You can use square brackets around identifiers to link to the position. For example, `[hostTrustConfiguration]`.